CZ/pyranda_rayleigh_taylor/rayleigh_taylor_lc_build_venv_sim.yaml

```diff
@@ -78,32 +44,25 @@ study:
                                                            --random-velocity-magnitude $(RANVEL) \
                                                            --viz-freq-cycle $(VIZ_FREQ_CYCLE) \
                                                            --dump-freq-cycle $(DUMP_FREQ_CYCLE)
-          #depends: [make-env-lc]
           procs: 1
           walltime: "00:10:00"
 
     - name: post-process-simulation
       description: Post process individual simulation outputs
       run:
           cmd: |
-            echo "Placeholder for extra post-processing"
-            echo "Parent workspace: $(run-pyranda.workspace)"
+            echo "Ingest sina file into store"
+            sina ingest -d $(STORE) $(run-pyranda.workspace)/sina.json
             
           depends: [run-pyranda]
-          procs: 1
-          walltime: "00:10:00"
 
     - name: post-process-all
-      description: Post process all simulation outputs
+      description: Plots and GP 1
       run:
           cmd: |
-            # NOTE: may only want this one vs this + individual given simplicity of outputs?
-            echo "Placeholder for extra collective post-processing step"
-            echo "Parent workspace: $(post-process-simulation.workspace)"
+            $(PYTHON) $(RAYLEIGH_TAYLOR_PLOTTER) --name=$(basename $(SPECROOT)) --store $(STORE)
             
-          depends: [post-process-simulation]
-          procs: 1
-          walltime: "00:10:00"
+          depends: [post-process-simulation*]
```

Comment:
think this should be `_*` at the end of this depends instead of just `*`

Reply:
I don't think it matters because it works as is but it's probably best practice to add the _ anyway. @jwhite242 do you recommend the `_`?